specification/communication/data-plane/MediaComposition/preview/2022-07-16-preview1/mediacomposition.json

```diff
@@ -0,0 +1,1180 @@
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "mediaCompositionId",
+            "description": "",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "400": {
+            "description": "Bad Request.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "401": {
+            "description": "Unauthorized.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "403": {
+            "description": "Forbidden.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "429": {
+            "description": "Too many requests.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Delete Media Composition": {
+            "$ref": "./examples/MediaComposition_DeleteMediaComposition.json"
+          }
+        }
+      }
+    },
+    "/mediaCompositions/{mediaCompositionId}/:start": {
+      "post": {
+        "tags": [
+          "MediaComposition"
+        ],
+        "summary": "Starts the composition",
+        "operationId": "MediaComposition_Start",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "mediaCompositionId",
+            "description": "",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CompositionStreamState"
+            }
+          },
+          "400": {
```

Comment:
It's not necessary to document all the return codes **IF** all they do is match the HTTP return codes--you can simply use a default error return. This is cleaner. 
However, if you are adding information that is specific to your service in the error code, then you can document those. (This applies to all of your operations.)